content/applications/productivity/calendar/google.rst

```diff
@@ -29,14 +29,14 @@ in, and store credentials. Click :guilabel:`New Project`.
 .. image:: google/new-api-project.png
    :alt: Create a new API project to store credentials.
 
-.. tip::
-   Give the API Project a clear name, like "Odoo Sync", so it can be easily identified.
+Give the API Project a clear name, like "Odoo Sync", so it can be easily identified. Then click the
```

Comment:
Lowercase "project", user input in literal formatting, and remove unnecessary adverbs please :) 
```suggestion
Give the API project a clear name, like `Odoo Sync`, so it can be identified. Then click the
```